packages/twenty-front/src/modules/object-record/record-board/types/AggregateContentId.ts

```diff
@@ -0,0 +1 @@
+export type AggregateContentId = 'aggregateOperations' | 'aggregateFields';
```

Comment:
I feel like "content" is too vague and could be more meaningful.

For example when we have tabs, tabId is meaningful, we understand that each tab has an id, but content is very broad and unless it's linked to another word it doesn't hold in itself the fact that we're talking about a dropdown menu content that can switch between multiple content.

Proposal, let's use more than one word to disambiguate : activeDropdownContentId and then we can be more specific for sub-dropdown modules : activeAggregateDropdownContentId (if needed)

Reply:
I can do it in my refactor PR